### PR TITLE
Added CNCF Landscape to Meshery Home page

### LINKED ISSUE #410: [Homepage] Adding CNCF Landscape to Meshery Home page
#### Current Behavior
We currently have no section highlighting Meshery as proudly representing service mesh management in the CNCF Landscape.

#### Desired Behavior
It will be good to created a new section, adding the CNCF landscape in Meshery home page

mesh-

Below is the design file 
https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI?node-id=1060%3A8

Below is the landscape svg
https://drive.google.com/file/d/1OtpsBtFesYT0FvpzlvK2KEqcJrgQC-k5/view?usp=sharing

### PR DESCRIPTION
**Pull request by Lezette**
Signed-off-by: Elizabeth Lola <leezeelola@gmail.com>

**Description**

This PR fixes #410 

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -321,6 +321,15 @@ <h5 style="text-align: center; color:aliceblue; font-size: large; ">
 		</h5>
 	</section>
 
+	
+	<section class="bg-white">
+		<div class="container" style="text-align: center;">
+			<h4 style="font-size: 2rem; line-height: 1.5; margin: 0; letter-spacing: 3px;">Meshery is in the CNCF Landscape</h4>
```
Comment (vinayaksh42) [inline index.html:327]: It will be a good practice if you move the in-line styling to its respective styling file.

**Review by vinayaksh42 — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (Lezette) [reply index.html]: Done! thank you

**Review by vinayaksh42 — APPROVED** [review]

**Comment by vinayaksh42** [comment]
nice work @Lezette , we can merge this now @leecalcote

**Comment by leecalcote** [comment]
Thank you, @Lezette @vinayaksh42

**Pull request merged**